api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -160,5 +236,36 @@ def update_app_list
         end
       end
     end
+
+    # TODO: Move to OrganizationCsvValidator
+    def validate_csv(csv)
+      errors = []
+      missing_headers = REQUIRED_HEADERS - csv.headers
+      if (missing_headers.any?)
+        errors.<< "Headers are missing: #{missing_headers.to_sentence}"
+        return errors
+      end
+      index = 1
+      csv.each do |row|
+        MANDATORY_COLUMNS.each do |c|
+          errors << "Row: #{index}, Missing value for column: ''#{c}''." if row[c].blank?
+        end
+        email = row['email']
+        if email.present? && Devise.email_regexp
+          errors << "Row: #{index}, Invalid email: ''#{email}''" unless email =~ Devise.email_regexp
+        end
+        password = row['password']
+        if password.present?
+          if Devise.password_regex && password !~ Devise.password_regex
+            password << "Row: #{index}, Invalid password'"
+          end
+          errors << "Row: #{index}, Invalid password, It's too short, minimum length is 6." if password.length < 6
```

Comment:
Default value of MnoHUb

Reply:
Can you put this magic number in a constant?